packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingEditorState.ts

```diff
@@ -422,11 +420,11 @@ const getMappingEmbeddedSetImplementations = (
     .map(getEmbeddedSetImplementations)
     .flat();
 
-/* @MARKER: NEW CLASS MAPPING TYPE SUPPORT --- consider adding class mapping type handler here whenever support for a new one is added to the app */
 const getMappingElementByTypeAndId = (
```

Comment:
This bit of logic looks really silly now.

I think the reason why we ever introduced the concept of `MAPPING_ELEMENT_SOURCE_ID_LABEL` was to properly identify the class mapping, but I don't think that's needed. We can basically remove all `MAPPING_ELEMENT_SOURCE_ID_LABEL.X` and replace them by `MAPPING_ELEMENT_TYPE.CLASS` when constructing the `lambaId`.

Also, I think

```ts
return (
        getAllClassMappings(mapping).find(
          (classMapping) => classMapping.id.value === id,
        ) ??
        getMappingEmbeddedSetImplementations(mapping)
          .filter(filterByType(EmbeddedFlatDataPropertyMapping))
          .find((me) => me.id.value === id)
      );
```

could be used for all `case MAPPING_ELEMENT_TYPE.CLASS:` So we technically don't need the plugin method here.